bioimageio_spec_version: 0.4.8.post1
rdf_sha256: 0170669407f703cb5db80459f84768a643d45c76651f2c6abcf6a36bbdbb2f5f
status: failed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.8post1
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.8 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags for categories: [{''network'': [''unet'', ''densenet'',
        ''resnet'', ''inception'', ''shufflenet'']}]'}
  - bioimageio_spec_version: 0.4.8post1
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.8
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags for categories: [{''network'': [''unet'', ''densenet'',
        ''resnet'', ''inception'', ''shufflenet'']}]'}
  - error: No module named 'tqdm'
    name: import test_resource from test environment
    status: failed
    traceback: ["  File \"scripts/dynamic_validation.py\", line 41, in main\n    from
        bioimageio.core.resource_tests import test_resource\n", "  File \"/home/runner/micromamba-root/envs/6559475/lib/python3.7/site-packages/bioimageio/core/__init__.py\",
        line 13, in <module>\n    from .prediction_pipeline import create_prediction_pipeline\n",
      "  File \"/home/runner/micromamba-root/envs/6559475/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/__init__.py\",
        line 2, in <module>\n    from ._prediction_pipeline import PredictionPipeline,
        create_prediction_pipeline\n", "  File \"/home/runner/micromamba-root/envs/6559475/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 12, in <module>\n    from ._stat_state import StatsState\n", "  File
        \"/home/runner/micromamba-root/envs/6559475/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_stat_state.py\",
        line 3, in <module>\n    from tqdm import tqdm\n"]
  deepimagej:
  - {bioimageio_spec_version: 0.4.8post1, error: Error while running DeepImageJ headless
      (CI did not produce an output image), name: Reproduce test outputs with DeepImageJ
      headless, status: failed}
  ilastik: []
